### DISCUSSION TITLE
Request: option to watch certain folder for gpx files

### DISCUSSION THREAD

**Discussion by el-tiuri** [Freika/dawarich#207] [Ideas]
Some applications, like Phonetrack, allow exporting gpx files automatically, so it’d be nice to have an option to monitor a folder for new gpx files instead of having to upload them manually.

**Comment by radh21301** [comment]
Maybe related to this:
https://github.com/Freika/dawarich/discussions/70

**Reply by Freika** [reply]
It's done in https://github.com/Freika/dawarich/releases/tag/0.15.0

**Reply by Ullas-Aithal** [reply]
> It's done in https://github.com/Freika/dawarich/releases/tag/0.15.0

Is it possible to add instructions on how to map the path in docker? I've been trying to map my local path to container paths `/var/app/public/imports/watched` , `/public/imports/watched/` , `/tmp/imports/watched/` and none of them worked for me :(

**Reply by el-tiuri** [reply]
Yeah, I couldn’t get it to work either. Really excited to see this implemented either way, let’s hope this is just a mistake on our part.

**Reply by Freika** [reply]
In 0.15.3 I reworked it a bit and you can find the instruction in the release notes: https://github.com/Freika/dawarich/releases/tag/0.15.3

**Reply by Ullas-Aithal** [reply]
That worked. Thank you for the efforts!
